src/mcts/node.h

```diff
@@ -177,16 +174,25 @@ class Node {
   // * N-in-flight (-=1)
   void FinalizeScoreUpdate(float v);
 
-  // Updates max depth, if new depth is larger.
-  void UpdateMaxDepth(int depth);
+  // Depth tracking and retrieval functions.
+  void UpdateMaxDepth(const uint16_t& depth) {
```

Comment:
Why pass by reference? As is it compiles to void UpdateAverageDepth(unsigned short const & __ptr64)
versus void UpdateAverageDepth(const uint16_t depth) which compile to void UpdateAverageDepth(unsigned short)